linux/deploy_linux.sh

```diff
@@ -1,11 +1,13 @@
-#!/bin/sh
```

Comment:
I'm not even sure who uses this script.

Reply:
Probably the old Sourceforce distro.  Pre-`.deb`.  Basically it's the "flavourless" Linux option.

Reply:
> I'm not even sure who uses this script.

I wondered about that as well. Personally, I'd say, delete it, but maybe others have more information about the history.

Reply:
Does anything else use the 
> `make dist`

artifacts?

Anyway, in a similar way to having a "Linux .deb" distribution, this provides a "Linux .tar.gz" distribution - but with no dependency information embeded (as it's not a .deb or .rpm, etc).

Reply:
Just tried it - it seems to include the source - I don't think we're planning to release the tar anymore. Linux users will be able to just git clone the repo.

Reply:
Hm, I'd be inclined just to delete it with a note in the changelog to that effect, then.

Reply:
Ok. Closing this issue then.